python_ta/debug/accumulation_table.py

```diff
@@ -144,15 +152,33 @@ def _tabulate_data(self) -> None:
             disable_numparse=True,
             missingval="None",
         )
-        if self.output_filepath is None:
-            print(table)
+
+        if self.output_format == "table":
+            if self.output_filepath is None:
+                print(table)
+            else:
+                try:
+                    with open(self.output_filepath, "a") as file:
+                        file.write(table)
+                        file.write("\n")
+                except OSError as e:
+                    print(f"Error writing table formatted data to file: {e}")
```

Comment:
You can keep the original error message in both branches; the message doesn't need to provide information about the output format.